eos-gpt2mbr

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash -e
+# Copyright (C) 2015 Endless Mobile, Inc.
+# Licensed under the GPLv2
+
+get_last_char() {
+	ret=$1
+	while [ "${#ret}" != "1" ]; do
+		ret=${ret#?}
+	done
+	echo $ret
+}
+
+# arguments checking
+if [ $# -lt 1 ]; then
+	echo "Usage: $0 <EOS partition>"
+       	exit 0
```

Comment:
Yes, but I followed the same convention as in `dracut/repartition/endless-repartition.sh`.